Web/Github/git-repo-stars.1h.sh

```diff
@@ -0,0 +1,118 @@
+#!/usr/local/bin/node
+
+/* Github repos stars count and Emoji on change :)
```

Comment:
Hey @softvar - again, for this you'll want to add some metadata. At least title, version, description & image please (https://github.com/matryer/bitbar#metadata for reference)